global.json

```diff
@@ -1,6 +1,6 @@
 ﻿{
   "sdk": {
-    "version": "6.0.100",
+    "version": "7.0.100",
```

Comment:
If we go ahead with my suggestion of "you only need .NET 7 to run the AOT test project", I assume we'd revert this but add another global.json in the AOT test project directory. Does that sound feasible?